src/error.rs

```diff
@@ -26,18 +26,26 @@ struct Custom {
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub enum ErrorKind {
     General,
+    EcsError,
```

Comment:
This variant is the only one with an "Error" suffix. Since the enum's name is already `ErrorKind`, this can be left out.

Reply:
Yeah, they all had suffixes. so I must have forgotten to remove it